### ISSUE TITLE
Add a11y tools

### ISSUE THREAD

**Issue by Verthon** [Verthon/portfolio#113]
* add automated check for a11y
* Cypress-Axe and axe-core
* add it to current github-pipelines to validate on mr
Eg.

```
import { mount } from '@cypress/react'; // For component tests
import 'cypress-axe';

it('Should have no accessibility violations', () => {
  mount(<YourComponent />);
  cy.injectAxe(); // Injects the axe-core library
  cy.checkA11y(); // Checks for accessibility violations
});
```
